R/value-box.R

```diff
@@ -6,7 +6,7 @@
 #' even a [shiny::plotOutput()]).
 #'
 #' @param title,value a [htmltools::tag()] child to display above `value`. If a
-#'   string a provided, it's automatically wrapped in a header tag.
+#'   string (or number) a provided, it's automatically wrapped in a header tag.
```

Comment:
wdyt about this?

```r
#' @param title,value A string, number, or [htmltools::tag()] to display as the
#'   title or value of the value box. The `title` appears above the `value`.
```

Unless you want to get into the default classes added to each item, I think

> automatically wrapped in a header tag

is slightly misleading. I think it's probably better to put string/number front and center.

Reply:
Yes, way better, thanks